package.json

```diff
@@ -3,10 +3,15 @@
   "version": "2.0.1",
   "description": "Server-less feature gates that don't compromise users' privacy.",
   "bin": "bin/user-gate.js",
+  "browser": "dist/bundle.min.js",
+  "files": [
+    "dist",
+    "src"
+  ],
   "main": "src/index.js",
   "scripts": {
     "test": "jasmine",
-    "build": "browserify src/gate/exports.js -o dist/bundle.js && browserify src/gate/exports.js -o dist/bundle.min.js -g uglifyify --debug",
+    "build": "browserify src/gate/exports.js -t [ babelify --presets [ @babel/preset-env ] ] -o dist/bundle.js && browserify src/gate/exports.js -t [ babelify --presets [ @babel/preset-env ] ] -o dist/bundle.min.js -g uglifyify --debug",
```

Comment:
Nice going including Babel! Not strictly required for our own purposes but a nice general gesture to previous users of the library as well as consistent with our other self-transpiling modules.

Reply:
Oh and yes uglify doesn't support ES6 out of the box, my bad :) Transpiling is one way to fix that, the other being to use `uglify-es`